acceptance/tests/facts/partitions.rb

```diff
@@ -13,7 +13,7 @@
       ['uuid', /^[-a-zA-Z0-9]+$/],
       ['partuuid', /^[-a-f0-9]+$/],
       ['mount', /^\/.*/],
-      ['label', /\w+/],
```

Comment:
Now that I think about it, it should probably just be /.*/, as it could be set to just about anything when its set.